sagemaker-clarify/computer_vision/object_detection/object_detection_clarify.ipynb

```diff
@@ -183,6 +183,7 @@
     ")  # We use this bucket to store model weights - don't hesitate to change.\n",
```

Comment:
<p>Please cite:</p><blockquote>Griffin, G. Holub, AD. Perona, P. The Caltech 256. Caltech Technical Report.	</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>